Add configurable paymentDays setting and update relevant logic

sawaneh1 opened the pull request:
## What are the changes about?
Provide a brief explanation of the changes you have made
-auto selecting date x days from the day of listing.
- Introduced a new paymentDays setting under System → Settings → Miscellaneous → Various choices.
- Commented out obsolete settings use_paperflow  and `integration_ebconnect`.
- Changes made in:
  - debitor/betalinger.php
  - systemdata/diverse.php
  - systemdata/sys_div_func.php
## Have you checked the following? 
- [x] I have updated relevant documentation at https://github.com/DANOSOFT/saldi/wiki
- [x] I have checked that i am not linking to any external resources such as external style- or javascript-files, that could compomise stabilaty
- [x] I have read and understood the developer guidelines  
      https://docs.google.com/document/d/1GOmomtvKf21OV2VWNOIPweDi4gJHpMCK5qXzrPrypUc/edit?usp=sharing

logicguy1 reviewed (commented):
What logic did you change? Can you point to the place in the code? Seems like you just formatted the files

sawaneh1 commented:
> What logic did you change? Can you point to the place in the code? Seems like you just formatted the files

Thank you for the feedback!

Here is a summary of the logic changes I made:

 In debitor/betalinger.php (around line 193), I added logic to automatically calculate the payment date using the paymentDays value from the settings.

    In systemdata/diverse.php (around line 236), I added:

$paymentDays = if_isset($_POST['paymentDays']);

This captures the paymentDays input when saving the setting.

In systemdata/sys_div_func.php (around line 734), I added:

    $paymentDays = get_settings_value("paymentDays", "payment", "");

    This retrieves the configured value from the system settings.

You can search for paymentDays in the code to see how it's used across the changes. Feel free to let me know if you need any further clarification.

sawaneh1 commented:
> I see, makes sense. Next time it is usually best to separate formatting and feature adding commits, otherwise we cant track your changes

You're right,  it was my code formatter in the editor that caused it. I will make sure to separate formatting and feature changes into different commits next time. Thanks for the feedback.
